packages/react/src/wallet/bitte-wallet.ts

```diff
@@ -57,15 +57,15 @@ export const BitteWalletAuth = {
   },
   setupBitteWalletSelector: async (
     callbackUrl,
-    onlyMbWallet = false,
+    onlyBitteWallet = false,
     network?,
     contractAddress?,
     options?: { additionalWallets?: Array<WalletModuleFactory> },
     successUrl?: string,
     failureUrl?: string,
   ): Promise<WalletSelectorComponents> => {
 
-    if (onlyMbWallet === false) {
+    if (onlyBitteWallet === false) {
```

Comment:
Not related to this Pr, but this entire if block is unnecessary and could probably be replaced with much more compact similar code as in react/WalletContext